prs_per_repo.rb

```diff
@@ -29,35 +29,40 @@ def merged?(repo, number)
     stats.client.pull_request(repo, number).merged?
   end
 
-  def pr_issues(repo)
+  def prs_since_sprint_start(repo)
     since = @sprint.range.begin.in_time_zone("US/Pacific").iso8601
     stats.pull_requests(repo, :state => :all, :since => since)
```

Comment:
`since` brings back too much information, and in the other merged_prs_per_sprint we use a range query instead which is much less data and is more accurate.  We probably should consolidate on that approach.

Reply:
If we do the range query only, we would miscount PRs remaining open, as we need to account for the PRs merged after a sprint ended.  But that can be obtained with another query, which is a more efficient approach.

Reply:
This is actually already handled .. we verified it in the merged_prs_per_sprint one